.gitignore

```diff
@@ -14,3 +14,7 @@
 .glide/
 
 auth.json
+/.DS_Store
```

Comment:
I'd prefer not to add platform-specific things (such as `.DS_Store`) here

Reply:
@nast90210 one great way to avoid adding this to a particular project is to use a global gitignore file just for your system (macOS in this case): https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
@powellc thx man - you advise is great!! I will try it!!

Reply:
done